core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -4721,24 +4551,132 @@ public DisplayType getDisplayType() {
     return mDisplayType;
   }
 
+  private static final Map<PropertyKey, Deprecated> DEPRECATED_KEYS =
+      populateAnnotatedKeyMap(PropertyKey.class, PropertyKey.class, Deprecated.class);
+  private static final Map<Template, Deprecated> DEPRECATED_TEMPLATES =
+      populateAnnotatedKeyMap(Template.class, Template.class, Deprecated.class);
+
+  private static final Map<PropertyKey, Removed> REMOVED_KEYS =
+      populateAnnotatedKeyMap(RemovedKey.class, PropertyKey.class, Removed.class);
+  private static final Map<Template, Removed> REMOVED_TEMPLATES =
+      populateAnnotatedKeyMap(Template.class, Template.class, Removed.class);
```

Comment:
this is a lot of initialization to require at classloading-time. Can we do this initialization lazily instead?